### ISSUE TITLE
Editing view needs to change for MacOS Mojave Dark mode

### ISSUE THREAD

**Issue by mirams** [opencor/opencor#1973]
I have MacOS Mojave 'Dark Mode' switched on which is nice, but the Editing window looks like this:!

<img width="983" alt="image" src="https://user-images.githubusercontent.com/858776/52265008-f65a7780-292a-11e9-8765-445d32c1a260.png">

**Comment by agarny** [comment]
Nothing has been done in OpenCOR about macOS Mojave's new "Dark" theme, but I agree that we ought to support, so tentatively putting it in for 0.7.

**Comment by agarny** [comment]
Having spent more time than I will ever want to admit on trying to get OpenCOR to look nice in Dark Mode, I have to admit "defeat". Qt is just not ready for Dark Mode despite [QTBUG-68891](https://bugreports.qt.io/browse/QTBUG-68891). So, we are just going to [enforce the use of Light Mode](https://developer.apple.com/documentation/appkit/nsappearancecustomization/choosing_a_specific_appearance_for_your_app) for now. Not something that I am happy about (not least because of all the time I wasted on this!), but too many other things to do, so...

**Comment by mirams** [comment]
Sorry Alan, enforcing light mode is fine! Thanks!!

**Comment by agarny** [comment]
> Sorry Alan, enforcing light mode is fine! Thanks!!

No, it's me. I should have given up long ago. I just thought that I could fix all the tiny problems I came across, but some of my fixes had side-effects, etc. due, I believe, to Qt not being ready for Dark Mode.

Anyway, the `master` branch now contains (the couple of lines of) code that enforces Light Mode, so I will generate a new snapshot before the end of the week.

**Comment by agarny** [comment]
When we first looked at this issue, we were using Qt 5.12.1. Right now, OpenCOR uses Qt 5.12.11 (definitely not the latest version of Qt, but this is unfortunately the last real LTS version) and it looks like dark mode support is quite good. So, we might be able to get it done this time.
